hack/fetch-releases.sh

```diff
@@ -265,6 +273,9 @@ main() {
 
   hub_version=$(${OPERATORTOOL} -config ${CONFIG} component-version hub)
   release_yaml_hub hub ${hub_version}
+
+  # copy pruner rbac/sa yaml
+  copy_pruner_yaml
```

Comment:
I dont know why we need this, we can add the config directly in kodata laso

Reply:
@piyush-garg pruner feature is available for both `kubernetes` and `openshift`. adding the common `yaml` into `kodata` can lead two entries(kubernetes/openshift). Hence I added it in a common place and moved it via script.